src/dashboard/src/components/access/views.py

```diff
@@ -464,7 +464,8 @@ def access_arrange_start_sip(client, request, mapping):
     parent = archival_object.get("parent", {}).get("ref")
     while parent:
         parent_obj = client.get_record(parent)
-        relation = [parent_obj["title"]] + relation
+        parent_title = parent_obj["title"] if parent_obj.get("title") else parent_obj["display_string"]
+        relation = [parent_title] + relation
```

Comment:
Thanks @djpillen this is neat an tidy. Only thing is we'll need you to run:

* https://github.com/psf/black

As you'll see CI currently fails on this snippet. 

You might also just be able to match the syntax which I think might be:
```
    parent_title = (
        parent_obj["title"] if parent_obj.get("title") else parent_obj["display_string"]
    )
    relation = [parent_title] + relation
```

But `black` is a good tool to have in your environment for future PR work and is in our `CONTRIBUTING.md`.

Once the CI passes I'll approve and we can merge.

Reply:
Just ran black and submitted a new commit.